src/Documentation/sidebar.json

```diff
@@ -43,6 +43,10 @@
         "label": "Data & Model Files Versioning",
         "slug": "data-and-model-files-versioning"
       },
+      {
+        "label": "Update a Tracked File",
+        "slug": "update-tracked-file"
```

Comment:
do you need a slug here?

Reply:
Will be changed to `update-tracked-files` in an upcoming commit anyway.

Reply:
Updated.